kubernetes/README.md

```diff
@@ -71,10 +72,77 @@ These are pause containers (docker_image:gcr.io/google_containers/pause.*) that
 
 The docker_daemon check ignores them through a default exclusion list, but they will show up for K8s metrics like *kubernetes.cpu.usage.total* and *kubernetes.filesystem.usage*.
 
+### How to elect a leader agent in order to avoid duplicated events?
+
+Since dd-agent 5.17, the agent has a leader election option for kubernetes.
+Every agent is a leader candidate, and they will perform a leader election among themselves using kubernetes components.
+The elected leader will be the only one responsible for collecting events.
+
+**This functionality is disabled by default**. 
+
+To enable it you need to set the variables `leader_candidate` to true in your kubernetes.yaml file.
+
+Internally it uses [ConfigMaps](https://kubernetes.io/docs/api-reference/v1.7/#configmap-v1-core) so you will need to get list, delete and create access to the ConfigMap resource.
```

Comment:
This feature relies on [ConfigMaps](https://kubernetes.io/docs/api-reference/v1.7/#configmap-v1-core), so you will need to grant Datadog Agent get, list, delete and create access to the ConfigMap resource.

Reply:
👍